runtime/stdlib/account.go

```diff
@@ -2769,6 +2829,29 @@ func getPathCapabilityControllerIDsIterator(
 	return
 }
 
+// AccountCapabilityStorageDomain is the storage domain which
+// records active account capability controller IDs
+const AccountCapabilityStorageDomain = "acc_cap"
```

Comment:
Similar to comment on previous PR, should test that this cannot escape to users.

Reply:
Added tests in a follow-up PR which addresses such PR feedback: [`fd60f78` (#2495)](https://github.com/onflow/cadence/pull/2495/commits/fd60f78532e32bfd38d7b343ac50f40c0f71971d)